ksqldb-examples/src/main/java/io/confluent/ksql/embedded/EmbeddedKsql.java

```diff
@@ -28,7 +29,8 @@ public static void main(final String[] args) {
 
     final KsqlContext ksqlContext = KsqlContext.create(
         new KsqlConfig(Collections.emptyMap()),
-        ProcessingLogContext.create());
+        ProcessingLogContext.create(),
+        new MetricCollectors());
```

Comment:
Hmm, this file is in `ksqldb-examples`, which raises the question of whether this is a public API. I can easily keep the old constructor for backward compatibility. Should I?

Also, is there anything else here that's a public API?